### PR TITLE
Accredited bodies round 2

### PR DESCRIPTION
**Pull request by jesseyuen**
Chipping away at these... this is another summary of Rachels research on accredited bodies from October last year. I'll think these will become far more once it's work I was involved in!

### CODE REVIEW

**Comment by fofr** [comment]
Please fixup or squash these commits.

File: app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+description: Continued research with Accredited bodies to determine reporting needs
+date: 2019-10-22
+related:
+  items:
+  - text: Lookback sessions
+    href: https://lookback.io/dfe-digital/find-accrediting-bodies-round-2
+  - text: Research playback
+    href: https://docs.google.com/presentation/d/1ChxmFgCA3MHBxtcuDwtkiUeNdYBXs2prmeqa9jtQuk8/edit#slide=id.g75687fc910_0_119
+---
+Round 2 was a continuation of the research started in [Accredited Bodies Research Round 1](link) where we hoped to gain further insights into the needs of Accredited Bodies around reporting.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md]: Link is missing

File: app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+## Findings
+
+### Student Information Systems
+
+Accredited bodies have a student record information system that syncs automatically with UCAS.
+
+[Lookback highlight](https://docs.google.com/presentation/d/1ChxmFgCA3MHBxtcuDwtkiUeNdYBXs2prmeqa9jtQuk8/edit#slide=id.g6b4896b14d_1_124)
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md]: Can these link directly to the clip on Lookback please

File: app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md [hunk 3]
```diff
@@ -9,63 +9,63 @@ related:
   - text: Research playback
     href: https://docs.google.com/presentation/d/1ChxmFgCA3MHBxtcuDwtkiUeNdYBXs2prmeqa9jtQuk8/edit#slide=id.g75687fc910_0_119
 ---
-Round 2 was a continuation of the research started in [Accredited Bodies Research Round 1](link) where we hoped to gain further insights into the needs of Accredited Bodies around reporting.
+Round 2 was a continuation of the research started in [Accredited Bodies Research Round 1](https://bat-design-history.netlify.com/publish-teacher-training-courses/accredited-bodies-research-round-1) where we hoped to gain further insights into the needs of accredited bodies around reporting.
```
Comment (fofr) [inline app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md]: Please use a relative link

File: app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md (hunk 2 above)
Comment (jesseyuen) [reply app/posts/publish-teacher-training-courses/2019-10-22-accredited-bodies-research-round-2.md]: I tracked down all but one of the clips in Lookback and have updated the missing Lookback link to read "Research Highlight" instead as it points to the playback.

**Comment by jesseyuen** [comment]
@fofr Guess who knows how to fix/squash commits now? ME. Ready for review... again.

**Review by fofr — APPROVED** [review]

**Pull request merged**